Mathlib/LinearAlgebra/Dual/Basis.lean

```diff
@@ -203,6 +203,15 @@ theorem eval_range {ι : Type*} [Finite ι] (b : Basis ι R M) :
     cases nonempty_fintype ι
     rw [← b.toDual_toDual, range_comp, b.toDual_range, Submodule.map_top, toDual_range _]
 
+lemma dualBasis_coord_toDualEquiv_apply [Finite ι] (i : ι) (f : M) :
+    b.dualBasis.coord i (b.toDualEquiv f) = b.coord i f := by
+  simp only [Basis.dualBasis, Basis.map_repr, LinearEquiv.trans_apply, coord_apply,
```

Comment:
Does just `simp` also work (without being much slower)? If so, can you use that, please?

Reply:
It does once one adds `-toDualEquiv_apply` - changed it to that.